File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Getting the Height of Mario's Pyramid
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Defining the line
    for (int a = 0; a < height; a++)
    {
        // Defining the first part of pyramid
        int b = 0;
        for (b = 0; b < height;)
        {
            // Moving one character forward to print
            b++;
            // Printing blank spaces
            if (height - a > b)
            {
                printf(" ");
            }
            // Printing #
            else
            {
                printf("#");
            }
            
        }
        // Printing the gap
        printf("  ");
        // Printing the second part of pyramid
        for (int c = 0; c < a + 1; c++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# Getting the Height of Mario's Pyramid
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")
        
    
# Defining the line
for a in range(height):
    for b in range(height):
        # Printing blank spaces
        if height - a > b + 1:
            print(" ", end="")
        # Printing blocks #
        else:
            print("#", end="")
    # Printing the gap
    print("  ", end="")
    # Printing the second part of Pyramid
    print("#" * (a + 1))